libsolidity/interface/StandardCompiler.cpp

```diff
@@ -155,6 +171,9 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 	if (!_input.isObject())
 		return formatFatalError("JSONError", "Input is not a JSON object.");
 
+	if (_input["language"] == "Julia")
+		return compileJulia(_input);
+
 	if (_input["language"] != "Solidity")
 		return formatFatalError("JSONError", "Only \"Solidity\" is supported as a language.");
```

Comment:
I thought you're going to catch that :)

The question is more about the Julia API.